server/app/views/applicant/TIAccountSettingsView.java

```diff
@@ -0,0 +1,144 @@
+import static j2html.TagCreator.td;
+import static j2html.TagCreator.th;
+import static j2html.TagCreator.thead;
+import static j2html.TagCreator.tr;
+
+import com.google.common.base.Strings;
+import com.google.inject.Inject;
+import com.typesafe.config.Config;
+import j2html.tags.specialized.DivTag;
+import j2html.tags.specialized.TdTag;
+import j2html.tags.specialized.TheadTag;
+import j2html.tags.specialized.TrTag;
+import java.util.Comparator;
+import java.util.stream.Collectors;
+import models.AccountModel;
+import models.TrustedIntermediaryGroupModel;
+import org.slf4j.LoggerFactory;
+import play.i18n.Messages;
+import play.mvc.Http;
+import play.twirl.api.Content;
+import services.MessageKey;
+import services.applicant.ApplicantPersonalInfo;
+import views.BaseHtmlView;
+import views.HtmlBundle;
+import views.admin.ti.TrustedIntermediaryGroupListView;
+import views.components.ToastMessage;
+import views.style.BaseStyles;
+import views.style.ReferenceClasses;
+
+public class TIAccountSettingsView extends BaseHtmlView {
+
+  private final ApplicantLayout layout;
+  private final String baseUrl;
+
+  @Inject
+  public TIAccountSettingsView(ApplicantLayout layout, Config configuration) {
+    this.layout = checkNotNull(layout);
+    this.baseUrl = checkNotNull(configuration).getString("base_url");
+  }
+
+  public Content render(
+      TrustedIntermediaryGroupModel tiGroup,
+      ApplicantPersonalInfo personalInfo,
+      Http.Request request,
+      Messages messages,
+      Long currentTisApplicantId) {
+    HtmlBundle bundle =
+        layout
+            .getBundle(request)
+            .setTitle("CiviForm")
+            .addMainContent(
+                h1(tiGroup.getName()).withClasses(BaseStyles.TI_HEADER_BAND_H1),
+                TrustedIntermediaryDashboardView.renderTabButtons(
+                    messages, baseUrl, TrustedIntermediaryDashboardView.TabType.ACCOUNT_SETTINGS),
+                div(
+                    renderSubHeader(messages.at(MessageKey.HEADER_ACCT_SETTING.getKeyName()))
+                        .withClasses(BaseStyles.TI_HEADER_BAND_H2),
+                    div(
+                            h3(messages.at(MessageKey.TITLE_ORG_MEMBERS.getKeyName()))
+                                .withClass("mt-8"),
+                            renderTIMembersTable(tiGroup, messages))
+                        .withClass("px-20")))
+            .addMainStyles("bg-white");
+
+    Http.Flash flash = request.flash();
+    if (flash.get("error").isPresent()) {
+      LoggerFactory.getLogger(TrustedIntermediaryGroupListView.class)
+          .info(request.flash().get("error").get());
+      bundle.addToastMessages(
+          ToastMessage.errorNonLocalized(flash.get("error").get()).setDuration(-1));
+    }
+    return layout.renderWithNav(request, personalInfo, messages, bundle, currentTisApplicantId);
+  }
+
+  private DivTag renderTIMembersTable(TrustedIntermediaryGroupModel tiGroup, Messages messages) {
```

Comment:
These methods were just moved from `TrustedIntermediaryDashboardView.java` to here without any changes.